refresh/fromswagger/swaggerize.js

```diff
@@ -184,10 +177,6 @@ function parseSwagger (api) {
     })
   } while (foundNewRef)
 
-  if (Object.keys(resources).length === 0) {
-    throw new Error('no resources')
-  }
-
```

Comment:
Yes, if there were no resources, then the validator will throw an error, so this code cannot ever be executed.